### DISCUSSION TITLE
Adding CreateDate and UpdateDate to Orchard Core Tables

### DISCUSSION THREAD

**Discussion by GregFinzer** [OrchardCMS/OrchardCore#10537] [Q&A]
I would like to add CreateDate and UpdateDate columns to all the Orchard Core Tables.  Will this hurt anything?  Will Orchard still work with the extra columns?

**Comment by hishamco** [comment]
I think AuditTrail module already did that (for auditing purpose), @Piedone correct me if I'm wrong

**Reply by Piedone** [reply]
If you want to rack this for content item changes and user operations then yes, see: https://www.youtube.com/watch?v=uMBN6TVhaMg

**Comment by ns8482e** [comment]
@GregFinzer  Do you mean to add to `Document` or On Index Tables?

**Comment by Skrypt** [comment]
CreatedUtc, ModifiedUtc, PublishedUtc already exists in the ContentItemIndex table. For everything else, not related to a content item, you should use Audit Trail. I don't think there's a need for this.
